tests/csm/rest/test_information.py

```diff
@@ -0,0 +1,211 @@
+#Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""
+Tests various operations on Cortx Information using REST API
+"""
+import logging
+from http import HTTPStatus
+from string import Template
+import pytest
+from commons import configmanager
+from commons import cortxlogging
+from commons.utils import assert_utils
+from libs.csm.csm_setup import CSMConfigsCheck
+from libs.csm.csm_interface import csm_api_factory
+from config import CSM_REST_CFG
+
+class TestCortxInformation():
+    """
+    Tests related to Cortx Information
+    """
+    @classmethod
+    def setup_class(cls):
+        """
+        This function will be invoked prior to each test case.
+        It will perform all prerequisite test steps if any.
+        """
+        cls.log = logging.getLogger(__name__)
+        cls.log.info("Initializing test setups")
+        cls.csm_conf = configmanager.get_config_wrapper(
+                        fpath="config/csm/test_rest_information.yaml")
+        cls.rest_resp_conf = configmanager.get_config_wrapper(
+            fpath="config/csm/rest_response_data.yaml")
+        cls.log.info("Ended test module setups")
+        cls.config = CSMConfigsCheck()
+        setup_ready = cls.config.check_predefined_s3account_present()
+        if not setup_ready:
+            setup_ready = cls.config.setup_csm_s3()
+        assert setup_ready
+        cls.created_iam_users = set()
```

Comment:
This is not used anywhere. Can u run pylint on ur files and make sure there is no pylint error and warning.
Line width we use is 100 and not 80 (default)